UI/.docker/docker-compose.env

```diff
@@ -5,7 +5,7 @@ KHEOPS_API_URL=https://demo.kheops.online/api
 #KHEOPS UI OIDC
 KHEOPS_OIDC_PROVIDER=https://keycloak.kheops.online/auth/realms/demo
 KHEOPS_UI_CLIENTID=loginConnect
-KHEOPS_UI_VIEWER_URL=https://ohif.kheops.online
+KHEOPS_UI_VIEWER_URL=https://ohif.kheops.online/viewer
```

Comment:
These changes just correspond to the earlier change to allow specifying which mode to run.